pysqldb3/geopackage.py

```diff
@@ -261,21 +265,35 @@ def shp_to_gpkg(self, shp_name, gpkg_tbl = None, print_cmd = False):
             gpkg_tbl = shp_name.replace('.shp', '')
 
         # check if the gpkg already exists. if so, we add another layer to the gpkg
-        if not self.geopackage_exists(self.gpkg_name):
+        if not self.gpkg_exists(self.gpkg_name):
```

Comment:
rather than repeating the cmd code, why not use the if then to set the format variable (which i think are the only things that are changing) and then just call it once - meaning just use the if/then to set the _overwrite and _update variables